lib/plugins/aws/lib/updateStack.js

```diff
@@ -94,6 +94,24 @@ module.exports = {
       });
   },
 
+  validateStackTemplate() {
+    const compiledTemplate = _.cloneDeep(
+      this.serverless.service.provider.compiledCloudFormationTemplate);
+
+    this.serverless.cli.log('Validating Stack...');
+    const params = {
+      TemplateBody: JSON.stringify(compiledTemplate),
+    };
+
+    return this.provider.request(
+      'CloudFormation',
+      'validateTemplate',
+      params,
+      this.options.stage,
+      this.options.region
+    ).then((cfData) => this.monitorStack('validate', cfData));
```

Comment:
I'm looking at `lib/plugins/aws/lib/updateStack.js` and it doesn't seem to define any list of actions anywhere. 

I'm also a bit confused by `monitorStack` itself - it seems to monitor the stack during creation, an update or deletion. Shouldn't this validation run _before_ the stack is created?